assets/js/admin/orders.js

```diff
@@ -11,4 +11,91 @@ jQuery( document ).ready( ( $ ) => {
 
 	'use strict';
 
+	const isCommerceOrder = Boolean( wc_facebook_commerce_orders.is_commerce_order );
+
+	window.WCFacebookCommerceOrderOperations = {
+		/**
+		 * Restrict order status options to only allowed options.
+		 *
+		 * @param {Object} $orderStatus Order select jQuery DOM object
+		 */
+		restrict_order_statuses             : $orderStatus => {
+
+			$orderStatus.find( 'option' ).each( function ( index, option ) {
+
+				// check if option value in the allowed list or not
+				if ( wc_facebook_commerce_orders.allowed_commerce_statuses.indexOf( option.value ) === -1 ) {
+					// delete/remove option if not allowed
+					option.remove();
+				}
+			} );
+		},
+		/**
+		 * Enable or Disable order created fields.
+		 *
+		 * @param {Boolean} enable wither to enable date fields (true) or not (false)
+		 */
+		toggle_created_date_fields_status   : enable => {
+			$( '#order_data' ).find( 'input[name*=order_date]' ).prop( 'disabled', !enable ).toggleClass( 'disabled', !enable );
+		},
+		/**
+		 * Disable order status field
+		 *
+		 * @param {Object} $orderStatus Order select jQuery DOM object
+		 */
+		disable_order_status_field          : ( $orderStatus ) => {
+			$orderStatus.prop( 'disabled', true ).addClass( 'disabled' );
+		},
+		/**
+		 * Toggle customer field
+		 *
+		 * @param {Boolean} hide
+		 */
+		toggle_order_customer_field         : ( hide ) => {
+			$( '#order_data' ).find( '.form-field.wc-customer-user' ).addClass( 'hidden', hide );
```

Comment:
I am not familiar with this second param to `addClass`. Will it only add the class if `hide` is true?